samples/benchmarks/coremark/boards/nrf54h20dk_nrf54h20_cpurad.conf

```diff
@@ -4,3 +4,13 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
 CONFIG_COREMARK_ITERATIONS=10000
+
+# Disable the UART console Kconfig to make Kconfig configuration with the DTS configuration.
+CONFIG_UART_CONSOLE=n
+
+# Kconfigs required for the STM standalone logging, imported from the nordic-log-stm snippet.
```

Comment:
Is there a reason why we cannot use the snippet?

Reply:
we do not support alternative logging configurations. The standard build command will fail either at build time or runtime

Reply:
Why does it fail?

Reply:
and later we won't have 4 comports

Reply:
we don't want to support more than one logging configuration per target to avoid maintenance cost

Reply:
I decided to stick with the approach without using snippets so that the simple target works without issues